docs/Lab4/README.md

```diff
@@ -1,12 +1,12 @@
-# Lab 4. I like sharing
+# Lab 4. Share Helm Charts
```

Comment:
Would have been nice to keep the old title .. less formal?

Reply:
same as above

Reply:
Replied above.